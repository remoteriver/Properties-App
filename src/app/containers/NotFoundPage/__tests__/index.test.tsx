import * as React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';

import { NotFoundPage } from '../index';

const renderer = createRenderer();

describe('<NotFoundPage />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<NotFoundPage />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
