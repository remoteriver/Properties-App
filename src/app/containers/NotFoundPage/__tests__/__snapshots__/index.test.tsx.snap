// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render and match the snapshot 1`] = `
<React.Fragment>
  <WithStyles(ForwardRef(Container))>
    <Styled(MuiBox)
      m={30}
    />
    <Styled(MuiBox)
      textAlign="center"
    >
      <WithStyles(ForwardRef(Typography))
        variant="h2"
      >
        404 Not Found
      </WithStyles(ForwardRef(Typography))>
    </Styled(MuiBox)>
  </WithStyles(ForwardRef(Container))>
</React.Fragment>
`;
